src/features/projects/components/PopupProject.tsx

```diff
@@ -42,7 +42,13 @@ export default function PopupProject({
   const projectDetails = project.properties;
 
   const handleDonationOpen = () => {
-    const url = getDonationUrl(project.properties.slug, token);
+    const url = getDonationUrl(
+      project.properties.slug,
+      token,
+      embed,
+      undefined,
```

Comment:
Please pass `callbackUrl` from the `ParamsContext` over here, like we're doing in `ProjectSnippet.tsx`